concepticondata/conceptrelations.tsv

```diff
@@ -452,12 +451,12 @@ SOURCE	SOURCE_GLOSS	RELATION	TARGET	TARGET_GLOSS
 2672	AUNT OR UNCLE	narrower	1327	UNCLE
 2683	EAGLE OR HAWK	narrower	1905	EAGLE
 2683	EAGLE OR HAWK	narrower	307	HAWK
-2686	CRUSTACEANS (PRAWNS OR SHRIMP)	narrower	238	SHRIMP
-2686	CRUSTACEANS (PRAWNS OR SHRIMP)	narrower	2349	PRAWNS
+2686	CRUSTACEANS (PRAWNS OR SHRIMP)	instanceof	238	SHRIMP
```

Comment:
"instanceof" is not handled by our internal algorithms. So please revert to original relation. this guarantees that we can assemble words for SHRIMP and PRAWNS, even if they are different. This is crucial for our concept list comparison algorithms and should be kept.

Same applies to goat/sheep. Watermelon is fine.

Reply:
So what exactly are the rules for when to use which relation? "Narrower" when there are only 2 or 3 possible hyperonyms, "instance of" when there is a basically unlimited amount?